PusherServer.Tests/AcceptanceTests/Trigger.cs

```diff
@@ -175,6 +176,71 @@ public void It_should_return_a_200_response_async()
     }
 
     [TestFixture]
+    public class When_Triggering_a_Batch_of_Events
+    {
+        [Test]
+        public void It_should_return_a_200_response()
+        {
+            IPusher pusher = new Pusher(Config.AppId, Config.AppKey, Config.AppSecret, new PusherOptions()
+            {
+                HostName = Config.Host
+            });
+
+            BatchEvent[] events = new BatchEvent[] {
+                new BatchEvent {
+                    channel = "my-channel-1",
+                    name = "my_event",
+                    data = new JsonSerializer().Serialize(new { hello = "world" })

```

Comment:
You should be able to make the `data` property an `object` from the API point of view. Not sure about how the internals work - but I'll take a look at the code in full.